src/confluent/azext_confluent/tests/latest/example_steps.py

```diff
@@ -0,0 +1,85 @@
+# --------------------------------------------------------------------------
```

Comment:
Tests are in two files. example_step.py, test_confluent_scenario.py. Is example_step.py just example code and should be deleted?

Reply:
The test is refactored in this way by codegen to better support both generated code and manual code for tests (you can add manual code to replace a step in the tests). `example_step.py` contains the actual operations generated from swagger test examples and is used in `test_confluent_scenario.py`.